### PR TITLE
feedflow 1.0.77 (new cask)

### PR DESCRIPTION
**Pull request by prof18**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/f/feedflow.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask "feedflow" do
+  version "1.0.77"
+  sha256 :no_check
```
Comment (khipp) [inline Casks/f/feedflow.rb:3]:
```suggestion
  sha256 "699b891da10e6dedd99956c159e444f79a24aa581e5c8c87a76ebd2dac044e84"
```

File: Casks/f/feedflow.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  url "https://github.com/prof18/feed-flow/releases/download/#{version}-desktop/FeedFlow-#{version}.dmg",
+      verified: "github.com/prof18/feed-flow/"
+  name "FeedFlow"
+  desc "Minimalistic RSS Reader"
```
Comment (khipp) [inline Casks/f/feedflow.rb:8]:
```suggestion
  desc "RSS reader"
```

The description should be concise (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).

File: Casks/f/feedflow.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      match[1]
+    end
+  end
+
+  depends_on macos: ">= :monterey"
+  depends_on arch: :arm64
+
+  app "FeedFlow.app"
+
+  zap trash: "~/Library/Application Support/FeedFlow"
```
Comment (khipp) [inline Casks/f/feedflow.rb:27]:
```suggestion
  zap trash: [
    "~/Library/Application Support/FeedFlow",
    "~/Library/Saved Application State/com.prof18.feedflow.savedState",
  ]
```

File: Casks/f/feedflow.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)[._-]desktop$/i)
+    strategy :github_latest do |json, regex|
+      match = json["tag_name"]&.match(regex)
+      next if match.blank?
+
+      match[1]
+    end
```
Comment (khipp) [inline Casks/f/feedflow.rb:19]:
```suggestion
    url :url
    strategy :github_latest
```

Will non-desktop releases be published in the GitHub repository, or will they continue to be tagged? If it's the latter, the suggestion above should suffice.

**Review by khipp — COMMENTED** [review]
Thank you for your first contribution to homebrew-cask, @prof18! 🎉 I left a few comments regarding some minor issues.

File: Casks/f/feedflow.rb (hunk 4 above)
Comment (prof18) [reply Casks/f/feedflow.rb:19]: Thanks!
There will be other tags but there will be only GitHub releases for desktop. So I guess the suggestion is fine?

Comment (prof18) [reply Casks/f/feedflow.rb:19]: For now, there are only desktop releases, but I'm planning to start adding releases for other platforms!

Comment (khipp) [reply Casks/f/feedflow.rb:19]: Then this suggestion should still be fine, granted you are manually setting the latest desktop release as `latest`.

Comment (prof18) [reply Casks/f/feedflow.rb:19]: Ah perfect, thanks!

**Review by khipp — APPROVED** [review]
Thank you!

**Pull request merged**